### DISCUSSION TITLE
（Oracle JDK 1.8.0_192） when run long time about 1 hour or more, agent multi process Exception

### DISCUSSION THREAD

**Discussion by popperjj** [naver/ngrinder#723] [Q&A]
（Oracle JDK 1.8.0_192）
when run long time about 1 hour or more,
agent multi process Exception:

99068 2020-12-28 23:00:15,991 INFO  ipt-ngrinder-prs-10-101-32-87.v-sz-1.vivo.lan-4: Report to console failed
99069 net.grinder.communication.CommunicationException: Exception whilst sending message
99070     at net.grinder.communication.AbstractSender.send(AbstractSender.java:57) ~[grinder-core-3.9.1.jar:na]
99071     at net.grinder.communication.QueuedSenderDecorator.flush(QueuedSenderDecorator.java:60) ~[grinder-core-3.9.1.jar:na]
99072     at net.grinder.engine.process.GrinderProcess.sendStatusMessage(GrinderProcess.java:563) ~[ngrinder-runtime-3.4.1.jar:na]
99073     at net.grinder.engine.process.GrinderProcess.access$1200(GrinderProcess.java:84) ~[ngrinder-runtime-3.4.1.jar:na]
99074     at net.grinder.engine.process.GrinderProcess$ReportToConsoleTimerTask.run(GrinderProcess.java:546) ~[ngrinder-runtime-3.4.1.jar:na]
99075     at java.util.TimerThread.mainLoop(Timer.java:555) [na:1.8.0_192]
99076     at java.util.TimerThread.run(Timer.java:505) [na:1.8.0_192]
99077 Caused by: java.net.SocketException: Broken pipe (Write failed)
99078     at java.net.SocketOutputStream.socketWrite0(Native Method) ~[na:1.8.0_192]
99079     at java.net.SocketOutputStream.socketWrite(SocketOutputStream.java:111) ~[na:1.8.0_192]
99080     at java.net.SocketOutputStream.write(SocketOutputStream.java:155) ~[na:1.8.0_192]
99081     at java.io.BufferedOutputStream.flushBuffer(BufferedOutputStream.java:82) ~[na:1.8.0_192]
99082     at java.io.BufferedOutputStream.flush(BufferedOutputStream.java:140) ~[na:1.8.0_192]
99083     at java.io.ObjectOutputStream$BlockDataOutputStream.flush(ObjectOutputStream.java:1823) ~[na:1.8.0_192]
99084     at java.io.ObjectOutputStream.flush(ObjectOutputStream.java:719) ~[na:1.8.0_192]
99085     at net.grinder.communication.AbstractSender.writeMessageToStream(AbstractSender.java:90) ~[grinder-core-3.9.1.jar:na]
99086     at net.grinder.communication.StreamSender.writeMessage(StreamSender.java:70) ~[grinder-core-3.9.1.jar:na]
99087     at net.grinder.communication.AbstractSender.send(AbstractSender.java:53) ~[grinder-core-3.9.1.jar:na]

**Comment by donggyu04** [comment]
@popperjj 

It seems socket exception while sending report data to console(web app)
Could you make a new issue comply with our template?

**Reply by popperjj** [reply]
done.
issue 724
